# Number of Robots
nRobots: 6

# Robot's radius
R: 0.5

# maximum robot's velocity [m/s]
vmax: 2.0

# Takeoff altitude [meters]
TOALT: 2.0

# origin and east GPS points
origin: [47.397742, 8.545594]
east: [47.3977419, 8.5458604]
h0: 535.273915611
